pkg/util/env/env.go

```diff
@@ -48,6 +50,9 @@ func GetNodeName() (string, error) {
 		klog.Errorf("Failed to get local hostname: %v", err)
 		return "", err
 	}
+	if runtime.GOOS == "windows" {
```

Comment:
would be good to have a comment here to explain why this is necessary on Windows